### DISCUSSION TITLE
Hideable Menubar

### DISCUSSION THREAD

**Discussion by AndreHaupt85** [neuefische/questions#162] [Web]
Hallo liebe Coder /innen,

ich beiße mir grade die Zähne daran aus eine Navbar zu schreiben die aber erst eingeblendet wird wenn man das Burgermenü klickt.

Guides die ich gefunden habe lassen die Menübar immer nur schrumpfen aber nicht verschwinden.
Außerdem benutzen alle React-Router-Dom.

Hat vielleicht jemand eine Idee oder kann mich in die richtige Richtung pushen?

Auf dem Bild sieht man rechts die normale Page und links die Navbar die bei klick auf das Burgermenu von rechts rein fahren soll.
![menu](https://user-images.githubusercontent.com/109967968/191773591-01117608-8248-4db6-a64d-87795381b81b.png)

Danke im voraus.

André

**Comment by ThomasKirst** [comment]
Hi André,

here you will find a guide which could help you implementing the burger menu:
–> https://css-tricks.com/hamburger-menu-with-a-side-of-react-hooks-and-styled-components/

The `theme` part as described in the tutorial together with the `React Context API` is not needed in your case.

I hope this helps!

– Thomas
